.travis.yml

```diff
@@ -3,13 +3,14 @@ language: julia
 os:
   - linux
 julia:
-  - 1.0
-  # - 1.1
+  - "1.0"
+  - "1.1"
+  - "1.2"
```

Comment:
Will this not result in a race-condition if multiple workers try to push?

Reply:
I tried it out. If multiple workers try to merge, the first will succeed. The subsequent workers will give an error and exit immediately. There's no timeout or anything like that.

Reply:
Okay. I think we should run tests on multiple julia, but probably good to pick one version for the merging.

Reply:
You can just check julia version in `.travis.yml` and exit unless it is 1.2 or something.

Reply:
Done!